staging/src/k8s.io/apiserver/pkg/server/options/admission.go

```diff
@@ -85,7 +86,7 @@ func NewAdmissionOptions() *AdmissionOptions {
 		// admission plugins. The apiserver always runs the validating ones
 		// after all the mutating ones, so their relative order in this list
 		// doesn't matter.
-		RecommendedPluginOrder: []string{lifecycle.PluginName, mutatingwebhook.PluginName, validatingwebhook.PluginName},
+		RecommendedPluginOrder: []string{lifecycle.PluginName, mutatingwebhook.PluginName, validatingwebhook.PluginName, cel.PluginName},
```

Comment:
I'm in favor of putting cel validation _before_ validating webhooks, not after.

This makes ensures we only call out to validating webhooks once all other validation has succeeded, which is faster for the case where there are failures for cel validation.